src/ReportWriter.cs

```diff
@@ -15,6 +15,12 @@ public static void Write(string reportPath, IEnumerable<FormattedFile> formatted
             var reportFilePath = GetReportFilePath(reportPath);
             var reportFolderPath = Path.GetDirectoryName(reportFilePath);
 
+            // If no folder path is specified to the report path, set it to "." (which is the current working directory)
+            if (string.Empty.Equals(reportFolderPath, StringComparison.OrdinalIgnoreCase))
+            {
+                reportFolderPath = ".";
```

Comment:
Perhaps, checking `!string.IsNullOrEmpty(reportFolderPath)` instead of `reportFolderPath != null` in the following if statement would be a more clear fix.

Reply:
I considered it, but went for the string empty comparison as there was a null check in the next if statement, and I didn't want to change that semantic (otherwise the null check would be irrelevant).
Can change though if you prefer ;-).

Reply:
Ok good idea. Didn't properly check if the variable was used apart from the check to create a directory (which isn't). Will change.